cg_hermes/config/tags.py

```diff
@@ -167,6 +167,7 @@
 MUTANT_SPECIFIC = {
     "mutant-config": {"description": "Config settings for mutant analysis"},
     "mutant-log": {"description": "SLURM log for mutant analysis"},
+    "software-versions": {"description": "Versions of software used in analysis"},
```

Comment:
I'm not sure if this is a mutant specific tag. Maybe in the future it can also be used by other tools to report software versions, so perhaps including it as part of `REPORTING_COMMON` and updating the` docs/all_tags.md` is a better option?

Reply:
I agree, this sounds like a better solution! Thanks!